### DISCUSSION TITLE
Search criteria in knowledge base, does not show results

### DISCUSSION THREAD

**Discussion by krseguros** [YetiForceCompany/YetiForceCRM#16765] [How to]
Hello,
I would like to know why when I try to search for an entry already created in the knowledge base, in the internal search engine, it does not find a match even for already registered entries, while in the module's search engine it does find the match:
1. YetiForce CRM 6.4 Service Pack v6 | 6.4.0
YetiForce - Latest stable
3. I checked several knowledge base entries, but I didn't find anything related
4. Run tests on https://gitstable.yetiforce.com/ and the same thing happens.
![image](https://user-images.githubusercontent.com/96985476/209582811-edfeff1a-c264-4f95-8bac-4955526d3a7e.png)
![image](https://user-images.githubusercontent.com/96985476/209582920-d6d87262-c08e-441c-97f3-80c3dcaab408.png)
Mi instancia crm
![image](https://user-images.githubusercontent.com/96985476/209583131-b4a85215-1603-438c-b0ed-93a4204132b2.png)
![image](https://user-images.githubusercontent.com/96985476/209583267-cbeedf26-0964-45b6-bacf-49ff2edb8ae8.png)

5. Current behavior (does not show results even though there are entries with the word used)
6. Behavior expected by your environment (showing the results and matches of the word used)

**Comment by apcloic** [comment]
@krseguros
Hi,
Did you enable cron ?
https://yetiforce.com/en/knowledge-base/documentation/administrator-documentation/item/enable-cron

**Reply by krseguros** [reply]
if it is active

**Reply by apcloic** [reply]
Sorry, I don't understand what do you mean by "if it is active"

1- Follow the instructions and make sure that cron is enabled and properly configured.
2- Go to Search configuration : https://gitstable.yetiforce.com/index.php?module=Search&parent=Settings&view=Index

That's it.

**Comment by krseguros** [comment]
If it is enabled, before asking the question, consult this, in fact to see if it changed the behavior I included in addition to the subject, content and purpose.
![image](https://user-images.githubusercontent.com/96985476/209661751-b01d77e0-a811-4edd-8294-85e3c129d2e3.png)
![image](https://user-images.githubusercontent.com/96985476/209662029-e0a52757-cfea-4a6b-94e9-7ef74beac05a.png)

**Reply by apcloic** [reply]
You have to wait for cron task to run (every 15min in gitstable) once you make some changes in search configuration, then search will work.
I don't see any issue with search function on gitstable.
![image](https://user-images.githubusercontent.com/2930560/209665841-c20daeeb-e29e-4f11-9133-c5d7689f66d0.png)
